docs/documentation/getting-started.md

```diff
@@ -0,0 +1,280 @@
+
+![saros consistency
+button](images/saros_consistency_button_0.png)**Inconsistency
+repair:**
+
+Sometimes accidents can happen and your copy of the project can become
+out of sync with the host's copy. When it does, this button will light
+up. When you click it, the inconsistencies will be repaired.
+
+![follow mode](images/followmode_0.png)**Switch
+Follow Mode on/off:**
+
+You can turn the Follow Mode on and of. In Follow Mode Saros will
+automatically sync your Eclipse view with the user you are following,
+opening shared files the user is opening, and automatically scroll to
+the visible part of the editor of open files, so you always see what the
+followed user sees.
+
+![saros leave button
+](images/saros_leave_button.png)**Leave the
+session:**
+
+Click here to leave the current session. If you are the
+[host](GettingStarted.md#Step%200:%20The%20Host), the session will be
+closed and all participants will be ejected from the session.
+
+#### b) Context Menu
+
+![write
+access](images/buddy_saros_obj.png)**Grant Write
+Access:**
+
+Gives the selected participant write access.
+
+![read-only
+access](images/participant_readonly_0.png)**Restrict
+to Read-Only Access:**
+
+Removes the write access right from the selected participant. The
+affected participant will only have read access afterwards.
+
+![follow mode
+icon](images/followmode_0.png)**Follow
+Participant:**
+
+Follows the selected participant as described in section [Follow
+Mode](GettingStarted.md#Good%20to%20Know:%20Follow_Mode).
+
+![jump icon](images/jump.png)**Jump to
+Position:**
+
+Jumps to the cursor position of the selected participant and open the
+corresponding file if necessary.
+
+**![open chat image](images/chat_misc.png) Open
+chat:**
+
+Opens a chat with this contact on the right side.
+
+![share file
+icon](images/saros_sharefile_button.png)**Send
+File:**
+
+Gives you the opportunity to select a file to be sent to the selected
+participant.
+
+### Good to know
+
+#### a) User Roles
+
+By default all participants of a session have **writing access**. To
+restrict or grant this access to other participants, right-click on that
+user in the session list and select *Restrict to read-only access* or
+*Grant writing access*. This can **only be done by the
+host**.
+
+#### b) Follow Mode
+
+You can use this feature to follow a single participant as s/he
+navigates the project and performs changes. When you follow a
+participant:
+
+-   Whenever s/he opens a file on his/her computer, it is opened on
+    yours too.
+-   Any time s/he switches to view a different file, it is switched
+    on yours.
+-   As s/he scrolls through a file, the viewpoint is moved on your
+    computer also, so that you see what s/he sees.
+
+#### c) Staying Aware of your Fellow Participants
+
+There are multiple ways of staying aware of what a driver is currently
+doing:
+
+-   In the package explorer (or resource navigator):
+    -   ![active file](images/active_file.png) A
```

Comment:
Unlike the other icons above, these two icons have a prominent border. 
![image 5](https://user-images.githubusercontent.com/4941859/42878689-37e8690a-8a8e-11e8-8363-f594d003bd96.png)